.github/workflows/check.yml

```diff
@@ -110,6 +110,27 @@ jobs:
           path: build/target/hale-studio-*macosx*.dmg
           retention-days: 14
 
+      - name: Set up Trivy
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y wget apt-transport-https gnupg lsb-release
+          wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | sudo apt-key add -
+          echo deb https://aquasecurity.github.io/trivy-repo/deb $(lsb_release -sc) main | sudo tee -a /etc/apt/sources.list.d/trivy.list
+          sudo apt-get update
+          sudo apt-get install -y trivy
+          trivy --version
+
+      - name: Run Trivy scan
+        run: |
+          trivy fs --severity CRITICAL,HIGH build/target/hale-studio-*linux*.tar.gz --format sarif --output trivy-results.sarif
+
+
+      - name: Upload Trivy scan results to file
```

Comment:
@stempler Do you maybe see why the file is not updated with the results?

Reply:
It seems the upload works but there are are just no results - I would guess maybe because the product archive is not extracted.

Would the goal not be to use the `github/codeql-action/upload-sarif` action again to integrate with GitHub code scanning?